packages/xo-web/src/xo-app/vm/tab-disks.js

```diff
@@ -476,6 +531,11 @@ export default class TabDisks extends Component {
     createCollectionWrapper(vdis => sortedUniq(map(vdis, '$SR').sort()))
   )
 
+  _getVdiByTaskId = createSelector(
+    () => this.props.vdis,
+    vdis => groupBy(vdis, "other_config['xo:import:task']")
```

Comment:
I don't think we should group tasks: in the future we may have multple task ( for exemple multiple exports )  on the same VDI